packages/transport/src/index.ts

```diff
@@ -32,3 +32,6 @@ export { SessionsBackground } from './sessions/background';
 export { SessionsClient } from './sessions/client';
 
 export { UdpTransport } from './transports/udp';
+
+// react-native only
+export { NativeUsbTransport } from './transports/nativeUsb';
```

Comment:
maybe it shouldn't be exported through index? 

I think it should be implemented like this

```
import { NativeUsbTransport } from '@trezor/transport/nativeUsb

TrezorConnect.init({
  transports: [new NativeUsbTransport()]
})
```

also please note that this is going to be the first usage when transport instance is passed to init, so there migh be problems. Let me know if you bump into something

Reply:
Why not? Generaly is not recommended to access anything in package that is not exported through index (same as private properties)